iwyu_output.cc

```diff
@@ -803,18 +804,29 @@ set<string> CalculateMinimalIncludes(
   }
 
   // Steps (2): Go through the needed private-includes that map to
-  // more than one public #include.  First choice: an include in
-  // associated_direct_includes (those are includes that are not going
-  // away, since we can't change associated files).  Second choice,
-  // includes in direct_includes that are also already in
-  // desired_headers.  Third choice, includes in desired_headers.
-  // Fourth choice, includes in direct_includes.  Picking in
-  // this order minimizes the number of #includes we add, while
-  // allowing us to remove #includes if need be.
+  // more than one public #include.  Use the following priority order:
+  // - Ourselves.
```

Comment:
I had trouble understanding why we prioritize the current header, until I read the rationale in the commit message/PR description. I think it would be nice to expand on the minimize-includes benefit here (if possible without too much verbiage).

Reply:
I feel that this is already covered by the sentence at the end of this comment block: "Picking in this order minimizes the number of #includes we add, while allowing us to remove #includes if need be."

Do you think more is needed?  Or perhaps it would be clearer if this were stated before the priority list?

Reply:
Yeah, reading it again, I think you're right. Ship it!